.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+    # See https://pre-commit.com for more information
+    # See https://pre-commit.com/hooks.html for more hooks
+    repos:
+      - repo: https://github.com/ambv/black
+        rev: 22.10.0
+        hooks:
+          - id: black
+            language_version: python3.10
+      -   repo: https://github.com/pre-commit/pre-commit-hooks
+          rev: v2.0.0
+          hooks:
+          - id: flake8
```

Comment:
can we add our own hooks in here? do we want to still include standards on commit messages?